test/utils.go

```diff
@@ -14,22 +14,22 @@ import (
 // const defines common image name
 var (
 	busyboxImage                string
+	busyboxImageID              string
+	busyboxImage125             string
```

Comment:
Does busybox 1.25 version has something special with busybox 1.28? Or why we can not  use just one version

Reply:
As @fuweid said, the 1.28 version alway update, but 1.25 version will not change, some test must keep the image isn't changed. @fuweid More reason can you explain？

Reply:
I think maybe it need to test digest or something, then how about give `1.25` more special name to make it different from `1.28`.

Reply:
> I think maybe it need to test digest or something, then how about give 1.25 more special name to make it different from 1.28.

Yes. The `busyboxImage125` is to be more specific here.